external/fv3fit/fv3fit/_shared/pytorch/test_graph.py

```diff
@@ -0,0 +1,117 @@
+import dataclasses
```

Comment:
Test files should go under `fv3fit/tests`.

Reply:
fixed